tests/test_image/test_io.py

```diff
@@ -65,6 +65,51 @@ def test_imread(self):
         with pytest.raises(TypeError):
             mmcv.imread(1)
 
+        # backend pillow
+        mmcv.use_backend('pillow')
+        self.gray_alpha_img_path = osp.join(self.data_dir, 'gray_alpha.png')
+        self.palette_img_path = osp.join(self.data_dir, 'palette.gif')
+        img_pil_grayscale1 = mmcv.imread(self.img_path, 'grayscale')
+        assert img_pil_grayscale1.shape == (300, 400)
+        img_pil_gray_alpha = mmcv.imread(self.gray_alpha_img_path, 'grayscale')
+        assert img_pil_gray_alpha.shape == (400, 500)
+        mean = img_pil_gray_alpha[300:, 400:].mean()
+        assert_allclose(img_pil_gray_alpha[300:, 400:] - mean, 0)
+        img_pil_gray_alpha = mmcv.imread(self.gray_alpha_img_path)
+        mean = img_pil_gray_alpha[300:, 400:].mean(axis=(0, 1))
+        assert_allclose(img_pil_gray_alpha[300:, 400:] - mean, 0)
+        assert img_pil_gray_alpha.shape == (400, 500, 3)
+        img_pil_gray_alpha = mmcv.imread(self.gray_alpha_img_path, 'unchanged')
+        assert img_pil_gray_alpha.shape == (400, 500, 2)
+        img_pil_palette = mmcv.imread(self.palette_img_path, 'grayscale')
+        assert img_pil_palette.shape == (300, 400)
+        img_pil_palette = mmcv.imread(self.palette_img_path)
+        assert img_pil_palette.shape == (300, 400, 3)
+        img_pil_palette = mmcv.imread(self.palette_img_path, 'unchanged')
```

Comment:
Is it equivalent to testing a static image with the palette, like that annotation for segmentation?

Reply:
Yes. The purpose here is to test image with palette format. But I did not find test data in this format, so I created one.